README.md

```diff
@@ -108,40 +108,63 @@ interface ExistingUser
 }
 ```
 
-### Trait: ExistingUserUniqueIdentifier
+### Trait: DbUniqueIdentifier
 
-The ```ExistingUserUniqueIdentifier``` trait provides a method ```uniqueIdentifier``` to fetch a user unique identifier (like a GUID) from either LDAP or your local database.
+The ```DbUniqueIdentifier``` trait provides a method ```uniqueIdentifier``` to fetch a user unique identifier (like a GUID) from your local database.
 
 ```
 <?php
```

Comment:
There is no need to include a complete class in your readme, it makes it hard to maintain two copies of the same file, and you'll probably forget to patch the readme; you can just refer them to the trait directly.

Reply:
removed class, updated text